src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
```

Comment:
it should be modifiers near `calendar bem block` here
<img width="726" alt="image" src="https://user-images.githubusercontent.com/102411130/223057598-a3d24c99-4cbd-4587-b3ab-081dffac82b5.png">